sys/src/regress/floatnotify.c

```diff
@@ -0,0 +1,27 @@
+#include <u.h>
```

Comment:
can you put a message in here about what this does, and a copyright notice? I don't understand what you're doing here.